### PR TITLE
CI: Journal-log is accessible and encrypted

### PR DESCRIPTION
**Pull request by lavocatt**

### CODE REVIEW

File: schutzbot/save_journal.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# use tee, otherwise shellcheck complains
+sudo journalctl --boot | tee journal-log >/dev/null
+
+# As it might contain sensitive information and is important for debugging
+# purposes, encrypt journal-log with a symmetric passphrase.
+gpg --batch --yes --passphrase "$GPG_SYMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```
Comment (achilleas-k) [inline schutzbot/save_journal.sh:8]:
```suggestion
gpg --batch --yes --passphrase "$GPG_SYMMETRIC_PASSPHRASE" -o journal-log.gpg --symmetric journal-log
```
A bit of a nitpick but if there's a typo in the variable name I'm sure it'll trip someone up in the future

**Review by achilleas-k — CHANGES_REQUESTED** [review]

File: schutzbot/save_journal.sh (hunk 1 above)
Comment (lavocatt) [reply schutzbot/save_journal.sh:8]: Indeed, good catch.

**Review by ondrejbudai — COMMENTED** [review]
Looks good, needs a rebase though.

**Review by achilleas-k — APPROVED** [review]

**Comment by msehnout** [comment]
Can we make the artifact disappear after some predefined time? In case the password ever leaked.

**Comment by jrusz** [comment]
We can set expiration time for artifacts https://docs.gitlab.com/ee/ci/pipelines/job_artifacts.html

**Comment by lavocatt** [comment]
The expiration is 30 days by default https://docs.gitlab.com/ee/user/admin_area/settings/continuous_integration.html#default-artifacts-expiration. If I understand the documentation correctly, the artifacts are kept indefinitely until a new job producing the same artifact runs successfully. Making the new artifact replacing the previous one. So if the password leaks, we should be able to fix the leakage by simply rerun the pipeline with a new password for the GPG key.

**Review by msehnout — APPROVED** [review]
Looks good :+1: 

I'd also like to see a proposal for key distribution and rotation, but preferably in a private channel, not on Github :-)

**Review by achilleas-k — APPROVED** [review]

**Pull request merged**